specification.md

```diff
@@ -460,8 +460,8 @@ and have the following action definition:
     {  
        "functionRef": "breadAndPastaTypesFunction",
        "actionDataFilter": {
-          "results": "${ [ .breads[0], .pasta[1] ] }",
-          "toStateData": "${ .itemsToBuyAtStore }"
+          "results": "${[ .breads[0], .pasta[1] ]}",
```

Comment:
this keeps the expression the same (just removes the leading trailing spaces). to keep it consistent to the changes you made for queries that return object type (removal of { } from valid expressions) and then keeping the array ones the same is confusing imo

Reply:
thx, same search/replace error.